pkg/ioutil/pagewriter.go

```diff
@@ -41,6 +41,9 @@ type PageWriter struct {
 // NewPageWriter creates a new PageWriter. pageBytes is the number of bytes
 // to write per page. pageOffset is the starting offset of io.Writer.
 func NewPageWriter(w io.Writer, pageBytes, pageOffset int) *PageWriter {
+	if pageBytes < 1 {
+		return nil
```

Comment:
Instead of returning `nil`, I think it makes more sense to add a [verification](https://github.com/etcd-io/etcd/blob/b7be91f98b7a7a6202ed9e9bc37b692d484585a8/client/pkg/verify/verify.go#L68), and panic when needed.

Reply:
The check is so inexpensive here, that it might be a regular assertion as well.

Reply:
I have turned the made the comparison a regular assertion, and it will panic instead of return nil when triggered

Reply:
Thank you. 
Thanks to @ahrtr we have a regular function for this: 
https://github.com/etcd-io/etcd/blob/29d30ab7b01f70375aa322fdbddb31a5acf7e003/client/pkg/verify/verify.go#L76

please use it.

Reply:
FYI. 
```
verify.Assert(pageBytes > 0, "pageBytes %d is an invalid value, it must be greater than 0", pageBytes)
```

Reply:
Thanks, I'll fix it